CHANGELOG.md

```diff
@@ -11,8 +11,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added support to `RimeHttpTTSService` for the `arcana` model.
 
+### Changed
+
+- Observers `on_push_frame()` now take a single argument `FramePushed` instead
+  of multiple arguments.
+
+### Deprecated
+
+- Observer `on_push_frame(src, dst, frame, direction, timestamp)` is now
+  deprecated, use `on_push_frame(data: FramePushed)` instead.
+
 ### Fixed
 
+- Fixed a `UltravoxSTTService` issue that would cause the service to generate
+  all tokens as one word.
+
```

Comment:
Just double checking if it should be here ?

Reply:
Yes, we forgot it in a previous PR.